src/main/scala/com/normation/rundeck/plugin/resources/rudder/RudderResourceModelSourceFactory.scala

```diff
@@ -115,8 +115,8 @@ object RudderResourceModelSourceFactory {
     .property(PropertyUtil.string(RUDDER_BASE_URL, "Rudder base URL"
       , "The URL to access to your Rudder, for ex.: 'https://my.company.com/rudder/'", true, null))
     .property(PropertyUtil.select(API_VERSION, "API version"
-      , "The API version to use for rundeck. For Rudder up to 5.0 use '6', for more recent version use '12'", true
-      , "latest", Seq("12", "6").asJava))
+      , "The API version to use for rundeck. For Rudder up to 5.0 use '6', for more recent version use '18'", true
+      , "latest", Seq("18", "12", "6").asJava))
```

Comment:
since we remove v6, we need to change that. I propose something like: 
```
      , "The API version to use for rundeck. You should use 'latest' appart for compat with older Rudder up to 7.x where version should be '12'", true
      , "latest", Seq("latest", "12").asJava))
```